changelogs/fragments/1021-git_config-custom-file.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - git_config - added parameter and scope ``file`` allowing user to change parameters in a custom file (https://github.com/ansible-collections/community.general/issues/1021).
+  - git_config - using list instead of string as first parameter in the ``run_command()`` call (https://github.com/ansible-collections/community.general/issues/1021).
```

Comment:
```suggestion
bugfixes:
  - git_config - using list instead of string as first parameter in the ``run_command()`` call (https://github.com/ansible-collections/community.general/issues/1021).
```

Reply:
I'd call it improvement - it was working before, so in theory not a "bug"